solr/core/src/java/org/apache/solr/handler/component/QueryElevationComponent.java

```diff
@@ -425,10 +429,10 @@ protected ElevationProvider loadElevationProvider(XmlConfigFile config) throws I
         previousElevationBuilder.merge(elevationBuilder);
       }
     }
-    return createElevationProvider(queryAnalyzer, elevationBuilderMap);
+    return createElevationProvider(elevationBuilderMap);
```

Comment:
Just checking: you did this because queryAnalyzer is in scope to caller so no need to pass as param?

Reply:
Yes